src/libraries/Microsoft.PowerFx.Interpreter/RecalcEngine.cs

```diff
@@ -214,7 +215,34 @@ public async Task<FormulaValue> EvalAsync(string expressionText, CancellationTok
             return result;
         }
 
-        // Preview functionality 
+        internal void AddUserDefinedFunction(string script, CultureInfo locale)

```

Comment:
Can we move to be on SymbolTable?
A practical scenario here is calling on Engine (not RecalcEngine) and demosntrating bind-only scenarios. 

Ok to have a helper on RecalcEngine that passes through to this.SymbolTable.